src/test/java/com/wikia/webdriver/common/core/api/ArticleContent.java

```diff
@@ -0,0 +1,70 @@
+package com.wikia.webdriver.common.core.api;
+
+import com.wikia.webdriver.common.core.TestContext;
+import com.wikia.webdriver.common.core.XMLReader;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Ludwik on 2015-08-05.
+ */
+public class ArticleContent extends ApiCall {
+
+  private static String secret;
+  private static String baseURL;
+
+  public ArticleContent() {
+  }
+
+  @Override protected String getURL() {
+    return null;
+  }
+
+  @Override protected User getUser() {
+    return null;
+  }
+
+  @Override protected List<NameValuePair> getParams() {
+    return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1168)